### ISSUE TITLE
Documentation example requires local modules

### ISSUE THREAD

**Issue by evetion** [aikubo/Linking-and-Clustering-Dikes#19]
Part of https://github.com/openjournals/joss-reviews/issues/6147

```python
import sys
sys.path.append("../src")
sys.path.append("/home/akh/myprojects/Dikes_Linking_Project/dikedata/spanish peaks")

# Packages written for the paper
from htMOD import HoughTransform
from clusterMod import AggCluster
from plotmod import DotsLines, plotScatterHist, pltRec, DotsHT
from PrePostProcess import *
from examineMod import examineClusters, checkoutCluster
from fitRadialCenters import RadialFit, NearCenters, CenterFunc
```

should probably become something like

```python
import linkinglines as ll

from ll.htMOD import HoughTransform
...  # etc
```

I believe a similar thing needs to be done for your tests to work automatically.

**Comment by nialov** [comment]
Same issues, I am getting the following error from running `pytest`:

~~~
============================= test session starts ==============================
platform linux -- Python 3.9.18, pytest-7.4.3, pluggy-1.3.0 -- /home/nialov/projects/LinkingLines/.venv/bin/python3-with-c-tooling
cachedir: .pytest_cache
rootdir: /home/nialov/projects/LinkingLines
configfile: pyproject.toml
plugins: anyio-4.2.0
collecting ... collected 0 items / 3 errors

==================================== ERRORS ====================================
__________________ ERROR collecting tests/clustering_test.py ___________________
ImportError while importing test module '/home/nialov/projects/LinkingLines/tests/clustering_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../nix-extra/result/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/clustering_test.py:12: in <module>
    from clusterMod import AggCluster
E   ModuleNotFoundError: No module named 'clusterMod'
________________ ERROR collecting tests/postProcessing_test.py _________________
ImportError while importing test module '/home/nialov/projects/LinkingLines/tests/postProcessing_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../nix-extra/result/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/postProcessing_test.py:7: in <module>
    from fitRectangle import rotateXYShift
E   ModuleNotFoundError: No module named 'fitRectangle'
_________________ ERROR collecting tests/preprocessing_test.py _________________
ImportError while importing test module '/home/nialov/projects/LinkingLines/tests/preprocessing_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../nix-extra/result/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/preprocessing_test.py:10: in <module>
    from htMOD import HoughTransform, CyclicAngleDist, rotateData, HT_center
E   ModuleNotFoundError: No module named 'htMOD'
=========================== short test summary info ============================
ERROR tests/clustering_test.py
ERROR tests/postProcessing_test.py
ERROR tests/preprocessing_test.py
!!!!!!!!!!!!!!!!!!! Interrupted: 3 errors during collection !!!!!!!!!!!!!!!!!!!!
============================== 3 errors in 2.37s ===============================
~~~

Following error from running the notebook:

~~~
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 20
     17 sys.path.append("/home/akh/myprojects/Dikes_Linking_Project/dikedata/spanish peaks")
     19 # Packages written for the paper
---> 20 from htMOD import HoughTransform
     21 from clusterMod import AggCluster
     22 from plotmod import DotsLines, plotScatterHist, pltRec, DotsHT

ModuleNotFoundError: No module named 'htMOD'
~~~

These issues make it difficult to verify the stated functionality of the package. Furthermore, the tests and notebook should use data that is either provided with the repository such as the already included `data/SpanishPeaks_3857.csv` or data that is downloaded during test initialisation. The first options is better but not suitable for larger files and binary data. Currently, I see use of absolute paths in the notebook:

~~~python
dikeset=pd.read_csv('/home/akh/myprojects/Dikes_Linking_Project/dikedata/spanish peaks/SpanishPeaks_3857.csv')
~~~

More robust use would be akin to the following:

~~~python
dikeset=pd.read_csv('data/SpanishPeaks_3857.csv')
~~~

**Comment by aikubo** [comment]
closed with #29
